Flat/src/ModulesLepPho.cc

```diff
@@ -9,139 +9,262 @@
 using namespace panda;
 using namespace std;
 
+void PandaAnalyzer::SimpleLeptons() {
+  looseLep1PdgId=-1, looseLep2PdgId=-1;
+  //electrons
+  for (auto& ele : event.electrons) {
+    float pt = ele.pt(); float eta = ele.eta(); float aeta = fabs(eta);
+    if (pt<10 || aeta>2.5) continue;
+    if (!ele.veto) continue;
+    if (!ElectronIP(ele.eta(),ele.dxy,ele.dz)) continue;
+    ele.setPtEtaPhiM(pt,eta,ele.phi(),511e-6);
```

Comment:
Also here?

Reply:
Same reason as for muons, the electron pT is not corrected in SimpleLeptons so this is not necessary